Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,123 @@
+# Hide shellcheck complaint:
+# shellcheck source=/dev/null
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
```

Comment:
I'm not sure what do you mean here. Noted, there will be three builds: OS X Intel, OS X PPC, Linux,